xml/System.Net.Http/HttpClient.xml

```diff
@@ -67,7 +67,29 @@ public class GoodController : ApiController
   
 ```  
   
-   
+The <xref:System.Net.Http.HttpClient> is a high-level API that wraps the lower-level functionality available on each platform where it runs.
+
+On each platform, <xref:System.Net.Http.HttpClient> tries to use the best available transport:
+
+| **Host/Runtime**  | **Backend**                                                                             |
+| ----------------- | --------------------------------------------------------------------------------------- |
+| Windows/.NET      | <xref:System.Net.HttpWebRequest>                                                        |
+| Windows/Mono      | <xref:System.Net.HttpWebRequest>                                                        |
+| Windows/UWP       | Windows native <xref:Syste.Net.Http.WinHttpHandler (HTTP 2.0 capable)                   |
```

Comment:
missing an m in System and closing tag for the xref link

Reply:
Thanks for the fixes!